### PR TITLE
Added sk translations

### PR DESCRIPTION
**Pull request by martinbarilik**
Hope everything is OK. If not, do not hasitate to make a note :) 

Here are my 2 sats to the cause.

### CODE REVIEW

File: client/app/components/Footer.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export async function Footer({ t, locale }: FooterProps) {
         <ul className="align-center flex flex-col justify-center gap-y-2 text-center sm:flex-row sm:flex-wrap sm:gap-x-8">
           {links.map(({ label, url }) => (
             <li key={label}>
-              <Link className="text-dark underline" href={url}>
+              <Link className="text-dark hover:text-dark" href={url}>
```
Comment (bitstein) [inline client/app/components/Footer.tsx:32]: Let's keep design oriented changes separate from this PR.

File: client/content/pages/sk/about.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+The Satoshi Nakamoto Institute was founded in November 2013 to advance and preserve knowledge of Bitcoin’s history, economics, and technology.
```
Comment (bitstein) [inline client/content/pages/sk/about.md:1]: Please remove any pages in `content/pages/sk/` that are still in English. i18next already defaults to the English version.

File: client/i18next-parser.config.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  locales: ["en"],
+  locales: ["en", "sk", "ko"],
```
Comment (bitstein) [inline client/i18next-parser.config.js:2]: We handle any translations via Crowdin, so please revert this change.

File: client/locales/sk/common.json [hunk 4]
```diff
@@ -0,0 +1,133 @@
+{
```
Comment (bitstein) [inline client/locales/sk/common.json:1]: Awesome!

**Review by bitstein — CHANGES_REQUESTED** [review]
I left some comments of some changes I would like on this. Please also fill out our translation community form so I can get you plugged in there, where we can discuss some further details before we bring this in. https://forms.gle/QbaYmkz1mJwBbErf6

Thank you so much, @martinbarilik! I look forward to getting a Slovakian translation of the website live.

**Comment by martinbarilik** [comment]
Thank you for your input. Sorry for including other commit ( honestly have no idea how that got included ).

Form filled. Hope you are satisfied with changes. 

Looking forward to contribute.

**Comment by bitstein** [comment]
@martinbarilik No problem, I'll clean this PR up when I have the chance.
